Improve team section in README

revolter opened the pull request:
Related to #11491.

With this:
- Clicking on the anchor next to a name, correctly jumps to the top of the image instead of the name
- Removes conflicting image size (`200` vs `140`)
- Replaces automatic link (added by GitHub) to the image file with a link to the GitHub profile

revolter committed:
Fix incorrect jump on anchor click

revolter committed:
Remove conflicting sizes for thumbnails

revolter committed:
Add link to GitHub profile

lacostej commented on Rakefile:
```diff
@@ -35,14 +35,19 @@ task(:generate_team_table) do
     content << "<tr>" if counter % number_of_rows == 0
-    content << "<td>"
-    content << "<img src='https://github.com/#{github_user}.png?size=200' width=140>"
+    content << "<td id='#{github_user_id}'>"
```
you could use github_user instead of constructing the id from the name.

revolter replied:
It doesn't work, because the anchor is automatically created by GitHub with that id as `href="#id"`. You can temporarily see this [here](https://github.com/revolter/fastlane/blob/693af3a54cedc2eb6830e5b59abfd6474b30e42a/README.md#j%C3%A9r%C3%B4me-lacoste) _(until I delete that branch)_.

revolter replied:
I also tried [here](https://github.com/revolter/fastlane/blob/656a4f005c134faad003e5a0980bf1ef8a709574/README.md#j%C3%A9r%C3%B4me-lacoste) to create that anchor myself, but GitHub still creates its own anchor.

revolter committed:
Add id for the team table

This assures that adding another table in the README doesn't get
replaced by the generate_team_table Rake task.

KrauseFx reviewed (approved):
Awesome, thanks 👍
